lib/EventsGenerator.js

```diff
@@ -27,14 +27,15 @@ class EventsGenerator extends ActionGenerator {
     // required
     this.option('config-path',
```

Comment:
only if low effort: rename `config-path => runtime-config-path` and same for `full-key-to-manifest`

Reply:
runtime action expects config-path. I am not sure if it will break other templates. So I will leave it as is